app/assets/javascripts/utilities/localDateTime.js

```diff
@@ -25,18 +25,44 @@ function timestampToLocalDateTime(timestamp, locale, options) {
   }
 }
 
-function addLocalizedDateTimeToElementsTitles(elements, timestampAttribute) {
+function timestampToLocalDateTimeLong(timestamp) {
```

Comment:
I have added this util function which uses ```timestampToLocalDateTime``` defined above to give longer version of date.

Reply:
I should have added these new functions at the end so that it would have shown better diff.